osu.Game.Rulesets.Taiko/Difficulty/TaikoDifficultyAttributes.cs

```diff
@@ -35,10 +35,16 @@ public class TaikoDifficultyAttributes : DifficultyAttributes
         public double ColourDifficulty { get; set; }
 
         /// <summary>
-        /// The difficulty corresponding to the hardest parts of the map.
+        /// Determines the amount of difficult strain peaks present within each skill.
         /// </summary>
-        [JsonProperty("peak_difficulty")]
-        public double PeakDifficulty { get; set; }
+        [JsonProperty("stamina_difficult_strains")]
+        public double StaminaTopStrains { get; set; }
+
+        [JsonProperty("rhythm_difficult_strains")]
+        public double RhythmTopStrains { get; set; }
+
+        [JsonProperty("colour_difficult_strains")]
+        public double ColourTopStrains { get; set; }
```

Comment:
Are we expecting to make imminent use of these? I'm fine with iterative changes but if these attributes aren't going to be used in a following PR then I'd rather not get them in yet to avoid unnecessary storage.

Reply:
Also, you're going to need to modify the database attribute methods to make sure that these new attributes are (de)serialized to/from database if they're going to be used in performance calculations.

Reply:
Imminent use, as well as this they won't be used in performance calculations, sr-only, thus I didn't write them to db